shared/src/main/kotlin/org/javacs/kt/util/ShellPathUtils.kt

```diff
@@ -33,3 +33,19 @@ private fun findExecutableOnPath(fileName: String): Path? {
 
     return null
 }
+
+fun findProjectCommandWithName(name : String, projectFile : Path) : Path? {
+	if(isOSWindows()) {
+		return findFileRelativeToProjectFile("$name.cmd", projectFile)
+	} else {
+		return findFileRelativeToProjectFile(name, projectFile)
+	}
+}
```

Comment:
Same as above, please also add a space before keywords like `if`. Also, simple functions like this can nicely be written as expressions:

```suggestion
fun findProjectCommandWithName(name: String, projectFile: Path): Path? =
    if (isOSWindows()) {
        findFileRelativeToProjectFile("$name.cmd", projectFile)
    } else {
        findFileRelativeToProjectFile(name, projectFile)
    }
```